Packs/ProofpointThreatResponse/README.md

```diff
@@ -1,3 +1,55 @@
 # Proofpoint Threat Response
+<~XSIAM>
+## Overview
+Proofpoint Threat Response is a security solution that helps organizations detect, analyze, and respond to threats quickly. It automates threat investigation and incident response to reduce risk and response time. By integrating with other security tools, it streamlines workflows and improves overall security operations.
 
-This pack includes Cortex XSIAM content.
+## This pack includes:
+- Rest API integration for security incidents
+- Modeling rules
+
+## Configure Proofpoint Threat Response Event Collector in Cortex XSIAM
+
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Server URL  | (e.g., https://192.168.0.1) | True |
+| API Key  | for the authentication. | True |
+| Trust any certificate (not secure) |  | False |
+| Use system proxy settings |  | False |
+| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | The time range for the initial data fetch. If timeout errors occur, consider changing this value. | False |
+| Fetch limit - maximum number of incidents per fetch |  | False |
+| Fetch delta - The delta time in each batch. e.g. 1 hour, 3 minutes. | The time range between create_after and created_before that is sent to the API when fetching older incidents. If timeout errors occur, consider changing this value. | False |
+| Fetch incidents with specific event sources. Can be a list of comma-separated values. |  | False |
```

Comment:
Is this parameter name **Fetch incidents with specific event sources** ?
The description should include: **Can be a list of comma-separated values.** 
But this doesn't actually describe the parameter, please provide more details